src/Language/PureScript/Docs/Prim.hs

```diff
@@ -322,20 +332,46 @@ number :: Declaration
 number = primType "Number" $ T.unlines
   [ "A double precision floating point number (IEEE 754)."
   , ""
-  , "Construct values of this type with literals:"
+  , "Construct values of this type with literals."
+  , "Negative literals must be wrapped in parenthesis if the negation sign could be mistaken"
+  , "for an infix operator (e.g. `negate`):"
```

Comment:
What does the ``(e.g. `negate`)`` mean here?

Reply:
`negate` is in reference to ~negate~ [`sub`/`-`](https://pursuit.purescript.org/packages/purescript-prelude/6.0.0/docs/Data.Ring#v:(-)). I think I'll just drop it.